src/get-started/java-binding.md

```diff
@@ -9,21 +9,46 @@ To run a node with your Java service you need to use Exonum Java application.
 You can download an archive containing the application and all the necessary
 dependencies on [the Releases page][github-releases] on GitHub. We suggest using
 `debug` version during development and `release` for deployment.
-Follow these steps:
 
-- Install [Maven 3][maven-install] which is essential for developing and building
-  Java service.
-- Install [Libsodium][libsodium] development library for your system.
-  You can use the following command for Ubuntu:
+### Mac OS
+
+For Mac users, we provide a [Homebrew][homebrew] repository, which gives the
+easiest way of installing Exonum Java App:
+
+```bash
+brew tap exonum/exonum
+brew install exonum-java
+```
+
+This will install `exonum-java` binary with all the necessary dependencies.
+However, you still need to install [Maven 3][maven-install] and follow the
+steps mentioned in [After Install][#after-install] section below.
+
+You can also install Exonum Java App manually:
```

Comment:
Do you think it shall be put in a separate section (so that it is clearer that one can skip it)? Or is it too heavyweight?